STLP : FAT filesystem file generated has incorrect name

**System information**
 - OS: Ubuntu 20.04
 - SDK version tag or commit SHA: 6fdb893f5a5394eb5dd78332741717f5c4c4032e

**Describe the problem**
When following the README for the STLP application, the step where the filesystem is flashed to the chip fails as below

```
[100%] Built target application_stlp
[100%] Create filesystem
Filesystem image application_stlp.fs successfully populated with contents of directory /tmp/tmp.UyLXOO3SwY
[100%] Flash filesystem
Error: F03146 Input file for option --data /home/angelcascarino/sandboxes/xcore_sdk/applications/stlp/filesystem_support/application_stlp_fat.fs not found
make[3]: *** [CMakeFiles/flash_fs_application_stlp.dir/build.make:71: CMakeFiles/flash_fs_application_stlp] Error 1
make[2]: *** [CMakeFiles/Makefile2:2416: CMakeFiles/flash_fs_application_stlp.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2423: CMakeFiles/flash_fs_application_stlp.dir/rule] Error 2
make: *** [Makefile:553: flash_fs_application_stlp] Error 2
```

On investigating the directory, a file named `application_stlp.fs` can be found. On renaming this to `application _stlp_fat.fs`, the installation can proceed as normal.

**Provide the exact sequence of commands / steps that you executed before running into the problem**

Steps as per the README.md in STLP application

**Other info or logs** 
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

--- a/applications/stlp/stlp.cmake
+++ b/applications/stlp/stlp.cmake
@@ -120,7 +120,7 @@ if(${CMAKE_HOST_SYSTEM_NAME} STREQUAL Windows)
 else()
     add_custom_command(
         OUTPUT application_stlp_fat.fs
-        COMMAND bash -c "tmp_dir=$(mktemp -d) && fat_mnt_dir=$tmp_dir && mkdir -p $fat_mnt_dir && mkdir $fat_mnt_dir/fs && cp ./demo.txt $fat_mnt_dir/fs/demo.txt && fatfs_mkimage --input=$tmp_dir --output=application_stlp.fs"
+        COMMAND bash -c "tmp_dir=$(mktemp -d) && fat_mnt_dir=$tmp_dir && mkdir -p $fat_mnt_dir && mkdir $fat_mnt_dir/fs && cp ./demo.txt $fat_mnt_dir/fs/demo.txt && fatfs_mkimage --input=$tmp_dir --output=application_stlp_fat.fs"
         DEPENDS application_stlp
         COMMENT
             "Create filesystem"

